projects/pt1/python/torch_mlir_e2e_test/configs/linalg_on_tensors_backend.py

```diff
@@ -32,7 +34,7 @@ def compile(self, program: torch.nn.Module) -> Any:
         example_args = convert_annotations_to_placeholders(program.forward)
         module = torchscript.compile(
             program, example_args, output_type="linalg-on-tensors")
-
+        logger.debug(module)
```

Comment:
Could add context to this log, unless there is already some provided by the caller.